src/codeql.test.ts

```diff
@@ -2,7 +2,7 @@ import fs from "fs";
 import path from "path";
 
 import { exec } from "@actions/exec";
-import * as io from "@actions/io";
+import io from "@actions/io";
```

Comment:
Doing this everywhere. Not sure why `import *` ever makes sense.

Reply:
I realised later that this was because the actions packages don't provide a `default` export, and so TBH I prefer the style of explicit imports in that case.

Reply:
Most of my inspiration is from https://github.com/github/codeql-action rather than any strong conviction.

I don't mind changing this kind of thing at all. Am I right in thinking that this particular change doesn't make any difference to code using the package?